Repair USE_NOBTCFI (not sure why it works, that still needs to be figured out).

Encrypted Media Extensions is not supported on OpenBSD. Apparently this also
means we can not pass --disable-eme to configure so comment that out.
Also see
https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/42501

Index: mozconfig-linux-x86_64
--- mozconfig-linux-x86_64.orig
+++ mozconfig-linux-x86_64
@@ -1,13 +1,7 @@
 . $topsrcdir/browser/config/mozconfigs/tor-browser
 
-# Moz switched to lld for all Linux targets in Bug 1839739.
-ac_add_options --enable-linker=lld
-
 ac_add_options --disable-strip
 ac_add_options --disable-install-strip
 
 ac_add_options --enable-default-toolkit=cairo-gtk3
-
-# Let's make sure no preference is enabling either Adobe's or Google's CDM.
-ac_add_options --disable-eme
 
